### PR TITLE
Updated vim syntax file for new gabc features.

### PR DESCRIPTION
**Pull request by henryso**
Since I use vim to edit pretty much everything, I decided to update the syntax file for most (maybe all) the new gabc features.

Please review and feel free to either merge it if you like it or discard it if you don't.

### CODE REVIEW

File: contrib/gabc.vim [hunk 1]
```diff
@@ -1,7 +1,7 @@
 " Vim syntax file
 " Language:	Gabc gregorian chant notation
 " Maintainer:	Elie Roux <elie.roux@telecom-bretagne.eu>
-" Last Change:	2008 Nov 29
+" Last Change:	2016 Feb 11
```
Comment (eroux) [inline contrib/gabc.vim]: Looks good, I trust you've tested it better than I would, the only remark is the "Maintainer" Line that should be changed or removed

Comment (henryso) [inline contrib/gabc.vim]: OK, I removed the "Maintainer" line.  I have tested this against most of the gabc files in the test repository.  I'm sure there are missing edge cases because of the looseness of the "syntax" of gabc, but I think it covers 90+% of cases.

**Comment by eroux** [comment]
Thanks!
